### ISSUE TITLE
CLI color support?

### ISSUE THREAD

**Issue by fritx** [typicode/hotel#76]
- colors
- progressbar

![image](https://cloud.githubusercontent.com/assets/6647633/14521408/c86a0064-025b-11e6-83d2-aef8d7aac494.png)

**Comment by typicode** [comment]
Hi @fritx 

Thank you for the feedback :)

1. colors (soon, there's a PR by @jalada, I'm working on subdomains right now and then I'll take a look at colors)
2. You mean progressbar in the CLI output like the one you have when you npm install for example?

**Comment by fritx** [comment]
@typicode yes. Is it possible to display the output of progressbar correctly?
It's so cool! I like this project! xD

**Comment by typicode** [comment]
@fritx I don't know actually, will check it.
Thank you :)

**Comment by typicode** [comment]
@fritx ANSI colors support has been released in `v0.4.19` :) (still need to check for progress)

**Comment by fritx** [comment]
@typicode that's amazing! will check it out soon :+1:

**Comment by typicode** [comment]
I've renamed it to progress bar since CLI colors are now supported.
